contributing.md

```diff
@@ -0,0 +1,39 @@
+# Contributing Guidelines
+
+This document outlines the process to update the docker images.
+
+## scheduling
+### For a recurring plugin update with no core
+This can happend anytime, but make sure not to bump within a security of LTS release period.
```

Comment:
```suggestion
## Scheduling

### For Plugin Update without Core Update

It can happen at anytime, but avoid merging during a security advisory or LTS Core release period.
```

For title capitalization: https://capitalizemytitle.com/style/Chicago/ is really useful